### DISCUSSION TITLE
Potential Calendar Enhancement?

### DISCUSSION THREAD

**Discussion by Livingnight** [nvim-telekasten/telekasten.nvim#162] [Ideas]
Hi all,

I'm really enjoying telekasten and the calendar and try to keep it open because I do a lot of writing and note taking with telekasten. One thing that would be nice, and hopefully easy to implement, would be a function that could close the calendar or make it so that it closes automatically when quitting nvim. I've done some searching and there doesn't seem to be a way to make it close like that when it's the last buffer open or to do it manually with a keymap. Please excuse me if I've just missed something obvious haha. Any help or ideas around this would be much appreciated!

![2022-09-15-13:43:15-screenshot](https://user-images.githubusercontent.com/26702985/190473923-437e4ea0-4991-4225-a94f-70443d2839ab.png)

![2022-09-15-13:43:22-screenshot](https://user-images.githubusercontent.com/26702985/190473942-2fd424ff-08d2-410e-b816-f0b945cc3ed3.png)

**Comment by lambtho12** [comment]
Makes sense.

But IMO this is more relevant either to the [original calendar-vim](https://github.com/mattn/calendar-vim) plugin or to [renerocksai's fork](https://github.com/renerocksai/calendar-vim) than in telekasten.
